app/src/dex/router/tests.rs

```diff
@@ -568,7 +568,7 @@ async fn fill_route_constraint_stacked() -> anyhow::Result<()> {
     let spill_price = U128x128::from(Amount::from(1_000_000_000u64) * pusd.unit_amount());
 
     let (unfilled, output) =
-        FillRoute::fill_route(&mut state_tx, delta_1, &route, Some(spill_price))
+        FillRoute2::fill_route(&mut state_tx, delta_1, &route, Some(spill_price))
```

Comment:
Oh, fyi the `fill_route_*` tests are skipped at the moment!

Reply:
Got it, I guess we should unskip them?